eng/MSBuild/LegacySupport.props

```diff
@@ -74,4 +74,8 @@
   <ItemGroup Condition="'$(InjectPlatformAttributesOnLegacy)' == 'true' AND ('$(TargetFramework)' == 'net462' or '$(TargetFramework)' == 'netstandard2.0' or '$(TargetFramework)' == 'netcoreapp3.1')">
     <Compile Include="$(MSBuildThisFileDirectory)\..\..\src\LegacySupport\PlatformAttributes\*.cs" LinkBase="LegacySupport\PlatformAttributes" />
   </ItemGroup>
+
+  <ItemGroup Condition="'$(InjectCollectionBuilderAttributesOnLegacy)' == 'true' AND ('$(TargetFramework)' == 'net462' or '$(TargetFramework)' == 'netstandard2.0' or '$(TargetFramework)' == 'netcoreapp3.1')">
```

Comment:
I realize this is just copying others, but, @joperezr, why are we still referencing netcoreapp3.1 anywhere in this repo?

Reply:
this could be a left-over artifact from the R9 repo.